src/openrct2-ui/windows/Player.cpp

```diff
@@ -475,7 +475,6 @@ void window_player_statistics_close(rct_window* w)
 {
     if (w->error.var_480)
     {
-        user_string_free(w->error.var_480);
```

Comment:
Is this sufficient, considering the rest of `var_480` does not appear to have been touched?

Reply:
I don't know where this code even came from. This variable is for the error window timer and has nothing to do with this custom window. It is not used anywhere else in this custom window, so I might as well delete this whole close method.